src/content/docs/style-guide/writing-docs/article-templates/whats-new-tech-writer-tasks.mdx

```diff
@@ -20,9 +20,16 @@ A high-altitude view of the `What's new` process looks like this:
 
 ## Specific tech writer tasks for posts [#specific-writing-tasks]
 
-Each day, the hero is responsible for checking the status of `What’s new` posts. The daily hero takes ownership of the post (reviewing, posting, and confirming that posts make it to docs site and product UI). When the post comes in, only assign it to yourself if you're going to see it through from start to finish. Otherwise, you can just monitor it as it goes through the workflow.
+The daily hero has the following responsibilities each day:
 
+* **Merge scheduled posts:** Check the `Scheduled work` column on the Github board for any `What's new` posts that have already been approved and are scheduled to merge that day. For example, if Monday's hero reviewed and approved a post scheduled to go live on your Wednsday hero shift, you should merge that PR.
+* **Review any incoming posts:** Any `What's new` posts that come in should be reviewed and approved during your hero shift. This includes basic copy edits, ensuring correct image formatting, and confirming a successful preview build. Edit the title to the day it should be published and then move the PR to the `Scheduled work` column. Reviewing PRs as they appear ensures a smooth release once they are ready to merge. 
```

Comment:
```suggestion
* **Review any incoming posts:** Any `What's new` posts that come in should be reviewed and approved during your hero shift. This includes basic copy edits, ensuring correct image formatting, and confirming a successful preview build. Edit the title to the day it should be published and then move the PR to the `Scheduled work` column. By reviewing PRs as they arrive, you ensure a smooth release once they are ready to merge. 
```

Reply:
@bradleycamacho you might add another comment like this: If you can't complete a "What's new" post review during your hero shift, please alert the hero taking the next shift so that they can complete it.